### ISSUE TITLE
Plans for the Ecosystem

### ISSUE THREAD

**Issue by segunadeleye** [nodejsnigeria/ecosystem#2]
I know we are already working on [one](https://docs.google.com/document/d/1tWM8MubZSpvyZAID4tTCB0eBbc4FYVmKU1KZLQRgRgs/edit)

We however need to conclude on the detailed plans for the ecosystem and steps to execute them.

It's important that we have this because it's what we would be selling to the attendees, potential contributors and potential sponsors.

**Comment by pmbanugo** [comment]
I think our next focus should be on training. I left a comment on the google sheet which I think we should follow up with and get that in that direction. Once we've gained momentum in the area of training/workshops, and seeing people starting to gain valuable insight, we can start on another one.

**Comment by OpesanyaAdebayo** [comment]
I think I agree with @pmbanugo. The next thing to figure out would be how the training should go. How frequently should it hold? Should we do: 

- 1-day training or a series?
- On-site training or remote?
- Free training or paid?

Since I'm asking the questions, I guess I should give my own answers first.

> How frequently should it hold?

In my opinion, I think we should do it quarterly (once in 3 months). The space affords us time to plan and improve, and also work on other things for the community.

> 1-day training or a series?

For now, it might not be sustainable meeting every week to train (even remotely) **so I think it's better if we do 1-day trainings quarterly**. 

> On-site training or remote?

**While I prefer on-site trainings for now**, I'll still list the pros and cons

**Pros of doing an on-site training:**
- It's easier for attendees to follow along.
- It's easier to interact with the audience.
- It removes issues such as poor internet connection.
- It makes it easier for us to generate PR assets because we can take pictures and record videos.
- It makes it easier to measure engagement.
- It removes internet cost of streaming, especially if it's a whole day training.
- It seems like the way to go if we are doing 1-day trainings.

**Cons of doing an on-site training:**
- It introduces venue cost: To manage this, I think we can:
  - Make it a paid (but affordable) training.
  - Put a cap on the audience (say 20-40 people max).
  - Use campus mini-conference halls. Since we'll first be starting in Lagos, because we might even be able to get venues free of charge if we partner with the CS groups on campuses.
- It introduces refreshments cost: This can be helped if we make the trainings paid.

- It introduces transportation cost: Well, I believe it's cheaper to pay for transport than to pay for internet for a whole day of video streaming.

> Free training or paid?

**I'll opt for paid training** because:
- It helps people stay committed
- It is easier to test interest level of the community
- It reduces the financial weight on the organizers

**In conclusion, my recommendation is: A 1-day, on-site, paid training program held quarterly.**

**Comment by segunadeleye** [comment]
Awesome. Training is definitely a good place to start.
We must however have a plan which of course goes beyond just the training.

**Comment by OpesanyaAdebayo** [comment]
Closing this, since we have moved discussions [here](https://github.com/nodejsnigeria/ecosystem/issues/4)
